src/ripple/app/paths/cursor/Liquidity.cpp

```diff
@@ -45,7 +45,7 @@ TER PathCursor::liquidity () const
             << "reverseLiquidity< "
             << "nodeIndex=" << pc.nodeIndex_
             << " resultCode=" << transToken (resultCode)
-            << " transferRate_=" << pc.node().transferRate_
+            << " transferRate_=" << *pc.node().transferRate_
```

Comment:
Problem is there's no `invalidRate` and I'd rather not add one. How about an `assert(static_cast<bool>(pc.node().transferRate_));` immediately after the `reverseLiquidity` call?